style.css

```diff
@@ -1,7 +1,10 @@
-@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@600&family=Pacifico&family=Poppins:wght@700&display=swap');
-/*font-family: 'Montserrat', sans-serif;
-font-family: 'Pacifico', cursive;
-font-family: 'Poppins', sans-serif;*/
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@800&family=Pacifico&family=Poppins:wght@700&display=swap');
```

Comment:
load fonts locally, if font resource will be down or be blocked you will not get fonts on site

Reply:
ok, I loaded fonts and link them locally by "@font-face". Should I delete "@import" or leave it on it's place and "@font-face" will be alternate option?

Reply:
if you go to this link you will see that you are importing font faces from remote, so use them locally and dont use remote

Reply:
Removed import link